src/states/Preroll.js

```diff
@@ -144,11 +135,12 @@ export default class Preroll extends AdState {
     // if there was an error.
     if (this.inAdBreak()) {
       player.ads.endLinearAdMode();
-    }
 
-    this.afterLoadStart(() => {
-      this.transitionTo(ContentPlayback);
-    });
+    } else {
```

Comment:
Does this change seem right @incompl? This way both endLinearAdMode and resumeAfterNoPreroll will wait for `playing` to transitionTo ContentPlayback

Reply:
adserror is deprecated and I think we should focus on removing reliance on it rather than fixing it.

Reply:
Cool, I think we agree that this change makes sense then?

Reply:
👍